leakcanary-android-sample/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,13 @@
     package="com.example.leakcanary"
     >
 
+  <uses-feature
+      android:name="android.software.leanback"
+      android:required="false" />
+  <uses-feature
+      android:name="android.hardware.touchscreen"
+      android:required="false" />
```

Comment:
Why is this here?

Reply:
These two are required for the app to be runnable on TV. 
I never tried running without these, though. Wonder if 'required' is not that strict of a requirement. Will try removing!

Reply:
If these are removed - the manifest starts to complain about missing features; although the app would still run on TV. They are needed, however, to make the app appear in the Play Store as available for TV:
https://developer.android.com/training/tv/start/start.html#leanback-req
So, while not necessary for our sample app, it's safe to keep it to make Lint happy.

What's more interesting is that I was wrong by adding these features to the `leakcanary-android-core`'s `AndroidManifest.xml`. Library doesn't have to define it, it's up to the users of the library to add TV support in their apps. I removed the features from that manifest file

Reply:
Nice! Ok I was confused as to why it was added in both places. That way is even better :)